scRNA-seq-advanced/setup/rms/Snakefile

```diff
@@ -1,29 +1,31 @@
-# Workflow for preprocessing SCPCP000005 data used for integration instruction
-# Raw files are expected to be present in the `raw/` directory inside `base_dir`
+# Workflow for preprocessing SCPCP000005 data used for integration and DGE instruction modules
+# Raw *_filtered.rds files from ScPCA are expected to be present in the `raw/` directory inside `base_dir`
 
 configfile: "config.yaml"
 outdir = config["base_dir"]
 
 rule target:
   input:
-    expand(os.path.join(outdir, "integration/{library}.rds"),
+    expand(os.path.join(outdir, "processed_libraries/{library}.rds"),
            library = config["libraries"])
 
 
-rule prepare_library:
+rule prepare_libraries:
   input:
     filtered_rds = os.path.join(outdir, "raw/{library}_filtered.rds"),
     celltype_tsv = config["celltypes_file"]
   output:
-    os.path.join(outdir, "integration/{library}.rds")
+    os.path.join(outdir, "processed_libraries/{library}.rds")
   log:
     "logs/{library}_prep.log"
   shell:
     """
-      Rscript prepare_integration_libraries.R \
+      Rscript prepare_rms_libraries.R \
```

Comment:
Since the script does only one library... (pedantic, yes) (similarly, I would have left the rule name singular, but it really doesn't mean anything)
```suggestion
      Rscript prepare_rms_library.R \
```